app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -48,6 +51,14 @@ def log_event_for_missing_threatmetrix_session_id
       analytics.idv_verify_info_missing_threatmetrix_session_id if idv_session.ssn_step_complete?
     end
 
+    def best_effort_phone
```

Comment:
Probably want test coverage for this method. I don't think we have a spec for `VerifyInfoConcern`, but you could either write one or put a test in the `VerifyInfoController` spec.